css/at-rules/container.json

```diff
@@ -36,6 +36,102 @@
             "deprecated": false
           }
         },
+        "anchor_position_queries": {
+          "__compat": {
+            "description": "Anchor position queries (`anchored()`)",
+            "spec_url": "https://drafts.csswg.org/css-anchor-position-2/#anchored-container-queries",
+            "support": {
+              "chrome": {
+                "version_added": "143"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "fallback": {
```

Comment:
Why are we documenting this (behavioral?) subfeatures?

Reply:
This is not a behavioral subfeature. It's a named container query "feature" (in media/container queries terminology, not the BCD sense), as in `@container anchored(fallback: <some-value>)`.

Right now there's only one feature to query against, but it's possible other container query features could exist (e.g., `@container anchored(foo-bar: <some-value>)` would get a `foo-bar` key as a peer to `fallback`).